cpp/oneapi/dal/test/engine/metrics/classification.hpp

```diff
@@ -16,22 +16,18 @@
 
 #pragma once
 
-#include <iostream>
-
 #include "oneapi/dal/test/engine/common.hpp"
 
 #include "oneapi/dal/table/row_accessor.hpp"
 #include "oneapi/dal/table/homogen.hpp"
 #include "oneapi/dal/table/detail/table_builder.hpp"
 
-namespace de = oneapi::dal::detail;
-
 namespace oneapi::dal::test::engine {
 
 template <typename Float = double>
-auto accuracy_score(const table& groundtruth,
-                    const table& prediction,
-                    const Float tolerance = Float(0)) {
+inline auto accuracy_score(const table& groundtruth,
```

Comment:
Why did you add `inline`?

Reply:
It's common practice to mark functions as inline in headers to prevent symbol's duplication